src/configs/javascript.mjs

```diff
@@ -0,0 +1,28 @@
+import esLint from "@eslint/js";
+import { config } from "typescript-eslint";
+import eslintPluginReact from "eslint-plugin-react";
+import importPlugin from "eslint-plugin-import";
+import unicornRules from "./providers/unicorn.mjs";
+import defaults from "./providers/default.mjs";
+import nextRules from "./providers/next.mjs";
+import { fixupConfigRules } from "@eslint/compat";
+
+import { FlatCompat } from "@eslint/eslintrc";
+
+const compat = new FlatCompat();
+
+const javascriptRules = fixupConfigRules(
+  config(
+    defaults,
+    esLint.configs.recommended,
+    unicornRules,
+    eslintPluginReact.configs.flat.recommended,
```

Comment:
unsure if we want to have react stuff in here? 
What if i have code that is non react that uses javascript? 

If nothing gets flagged by eslint it's fine for me to keep it in here.

Reply:
react seems to be limited to .tsx/.jsx.. so if there are jsx files, they will be linted by react rules.. at least I couldnt see any missflagged code yet :)